src/src/declarations.d.ts

```diff
@@ -17,6 +17,13 @@ import type { IR, RA, RR } from './utils/types';
 /* eslint-disable functional/prefer-readonly-type */
 
 declare global {
+
```

Comment:
This file is only for changing the type definitions of default javascript methods to make them more type safe.

Your custom typings should probably go into a different file. `./index.tsx` or `./Dashhboard/index.tsx` would be a good place for this at the moment.

Though, if you **really** want the type to be in global scope, then create a separate `.d.ts` file and put `WidgetObj` there